### ISSUE TITLE
Could you please provide the download link for the jointly masked pre-trained model in wmt？

### ISSUE THREAD

**Issue by yangkexin** [lemmonation/jm-nat#3]
Hi, this work I feel very interesting. If it is possible, could you please provide the download link in each data set on the optimal performance of the jointly masked model.pt files, I am eager to do some further test and the comparison between models, thank you :).

**Comment by lemmonation** [comment]
Thanks for your interest. Please refer to this [link](https://drive.google.com/file/d/18DNqFfoGNdf7htWBK_mEu3IwAHCmEFQn/view?usp=sharing) for wmt14 en-de ckpt and this [link](https://drive.google.com/file/d/1vLTgKDFh2zu4xxG6KRt7z32j0at-e37r/view?usp=sharing) for wmt14 de-en. We report BLEU scores computed by [compound_split_bleu.sh](https://github.com/pytorch/fairseq/blob/master/scripts/compound_split_bleu.sh).

**Comment by chynphh** [comment]
@lemmonation Thanks for sharing pre-trained models for wmt14 en<->de. If it is possible, could you please share the pre-trained model for  wmt16 en<->ro?
